packages/experiments/src/components/Chiclet/ChicletXsmall.noImage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { ChicletXsmall } from './ChicletXsmall';
+import { IChicletCardProps } from './ChicletCard.types';
+
+describe('Dialog', () => {
```

Comment:
Same above comment about this being 'Chiclet', which you only need to have once when you merge these tests into one file